command/job/hunt.go

```diff
@@ -0,0 +1,180 @@
+package job
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/nomad/api"
+)
+
+func Hunt() error {
+	// create Nomad API client
+	nomadClient, err := api.NewClient(api.DefaultConfig())
+	if err != nil {
+		return err
+	}
+
+	// Get the jobs
+	jobs, _, err := nomadClient.Jobs().List(nil)
+	if err != nil {
+		return err
+	}
+
+	for _, job := range jobs {
+		if job.Type != "service" {
+			continue
+		}
+
+		// Get job's running allocations
+		jobAllocations, _, err := nomadClient.Jobs().Allocations(job.ID, true, nil)
+		if err != nil {
+			return err
+		}
+
+		var firstRunningJobVersion uint64 = 0
+		for _, allocation := range jobAllocations {
+			// Find the first running job version
+			if allocation.ClientStatus == "running" && firstRunningJobVersion == 0 {
```

Comment:
if allocation.ClientStatus != "running"
continue

if firstRunningJobVersion == 0
set to job version

if allocation.JobVersion != firstRunningJobVersion
shame

makes it more readable if you break it up into this

Reply:
processing.....